src/main/groovy/nebula/plugin/dependencylock/utils/CoreLockingHelper.groovy

```diff
@@ -68,6 +71,17 @@ class CoreLockingHelper {
         project.plugins.withId("nebula.integtest") {
             runClosureOnConfigurations(configurationNames, closure, new ArrayList<String>())
         }
+        project.plugins.withType(JavaLibraryPlugin.class) {
+            runClosureOnConfigurations(configurationNames, closure, new ArrayList<String>())
+        }
+        project.plugins.withType(JavaPlugin.class) {
+            runClosureOnConfigurations(configurationNames, closure, new ArrayList<String>())
+        }
+        project.plugins.withType(JavaBasePlugin.class) {
```

Comment:
Will this part handle groovy, kotlin, clojure languages?

Reply:
Currently checking on the application of "nebula.kotlin". About to add a check for "nebula.clojure".

Reply:
Edit: checking on `KotlinBasePluginWrapper` rather than `nebula.kotlin`.